cos-fleetshard-operator-camel-it/src/test/resources/CamelConnectorReify.feature

```diff
@@ -38,6 +38,8 @@ Feature: Camel Connector Reify
           | trait.camel.apache.org/health.liveness-failure-threshold  | 6                                          |
           | trait.camel.apache.org/health.liveness-period             | 7                                          |
           | trait.camel.apache.org/health.liveness-timeout            | 8                                          |
+          | trait.camel.apache.org/health.prometheus.enabled          | true                                       |
```

Comment:
The name of the traits here are different from [here](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/pull/325/files#diff-6ef82b523452302e2a469ed1696df4eba39bb6de2e2ec55b98ba47aea9005a56R24-R25). 
Are are tests passing?

Reply:
The scenario fails

```
2022-04-13T08:03:59.1032910Z Expecting actual not to be null
2022-04-13T08:03:59.1033506Z 	at org.bf2.cos.fleetshard.it.cucumber.assertions.CucumberDataTableAssert.lambda$matches$0(CucumberDataTableAssert.java:36)
2022-04-13T08:03:59.1034167Z 	at java.base/java.util.LinkedHashMap.forEach(LinkedHashMap.java:684)
2022-04-13T08:03:59.1034662Z 	at java.base/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1505)
2022-04-13T08:03:59.1035332Z 	at org.bf2.cos.fleetshard.it.cucumber.assertions.CucumberDataTableAssert.matches(CucumberDataTableAssert.java:26)
2022-04-13T08:03:59.1036132Z 	at org.bf2.cos.fleetshard.operator.it.camel.glues.KameletBindingSteps.klb_annotation_contains(KameletBindingSteps.java:217)
2022-04-13T08:03:59.1037851Z 	at ✽.the klb has annotations containing:(classpath:CamelConnectorReify.feature:27)
```

But for some reason the test is not reported as failed

Reply:
Fixed